webpack/components/extensions/HostDetails/DetailsTabCards/ImageModeCard.js

```diff
@@ -28,6 +34,7 @@ const ImageModeCard = ({ isExpandedGlobal, hostDetails }) => {
       masonryLayout
       isExpandedGlobal={isExpandedGlobal}
     >
+      <a href={actionUrl(hostDetails.name)}>{__('Bootc action template')}</a>
```

Comment:
I think using "template" is a little confusing for users, perhaps we can find a title that better describes the relationship to running bootc actions on the host:
```suggestion
      <a href={actionUrl(hostDetails.name)}>{__('Modify via remote execution')}</a>
```

Reply:
@jeremylenz you mentioned having  some potential input too :)

Reply:
Right, I was thinking `Run bootc action` maybe? but actually I like this better!